jest.config.ts

```diff
@@ -21,4 +21,8 @@ module.exports = {
 	globalSetup: '<rootDir>/jest-global-setup.ts',
 	setupFilesAfterEnv: [ '<rootDir>/jest-setup.ts' ],
 	watchPlugins: [ 'jest-watch-typeahead/filename', 'jest-watch-typeahead/testname' ],
+	moduleNameMapper: {
+		'\\.css$': 'identity-obj-proxy',
+	},
+	transformIgnorePatterns: [ 'node_modules/(?!(highlight.js)/)' ],
```

Comment:
I interpret this to mean we need to transform the `highlight.js` source. Is that accurate?

Reply:
Correct, also related to the import of CSS referenced in [this comment](https://github.com/Automattic/studio/pull/196/files#r1623843319).